test/Tester/TestUtils.cs

```diff
@@ -114,20 +114,12 @@ public static class RequestContextTestUtils
     {
         public static void SetActivityId(Guid id)
         {
-#if NETSTANDARD
             RequestContext.ActivityId.Value = id;
```

Comment:
BTW, looking closer, should we just dump the ActivityId as a separate AsyncLocal field? Looks like we are putting it in the same dictionary as CallContextData when flowing it, and there is no inherent reason to have it separate. The reason some of it was separate, is that we were also augmenting and flowing the `Trace.CorrelationManager.ActivityId`, which is an external property that people might have already been using.

We can of course have a property in RequestContext so that people don't have to use a well-known header for it, such as:

```
        /// <summary>Gets or sets an activity ID that can be used for correlation</summary>
        public Guid ActivityId
        {
            get { return (Guid)(Get(E2_E_TRACING_ACTIVITY_ID_HEADER) ?? Guid.Empty); }
            set { Set(E2_E_TRACING_ACTIVITY_ID_HEADER, value); }
        }
```